templates/openstack/consulting.html

```diff
@@ -12,601 +18,564 @@
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <div class="p-section--shallow">
         <p>
-          <a href="/openstack/contact-us"
-             class="p-button--positive js-invoke-modal">Talk to the expert</a>
+          Once the design step is completed, we translate all design decisions into a model that is reusable across various environments. Modeling guarantees repeatable deployments and enables integration with infrastructure-as-code (IaC) and continuous integration continuous delivery (CICD) systems.
         </p>
       </div>
-      <div class="col-4 u-vertically-center u-hide--medium u-hide--small u-align--center">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-        alt="",
-        width="211",
-        height="150",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-        }}
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+      <h3 class="p-heading--5">3.&nbsp;&nbsp;Deploy</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <div class="p-section--shallow">
+        <p>
+          We deploy the cloud for you on your premises or in one of our partners&#39; data centers. We validate it post-deployment and benchmark its performance. At the end, we provide you with the deployment guide which outlines all the steps required to re-deploy the cloud if needed.
+        </p>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_4' -%}
+      <h3 class="p-heading--5">4.&nbsp;&nbsp;Operate</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_4' -%}
+      <div class="p-section--shallow">
+        <p>
+          After day-1, day-2 comes in. The cloud can either be managed by you or <a href="/openstack/managed">by our team of cloud experts</a>. In both cases we provide <a href="/openstack/support">24x7 commercial support</a>, including security updates, phone and ticket support, local regulatory compliance programs, etc.
+        </p>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_5' -%}
+      <h3 class="p-heading--5">5.&nbsp;&nbsp;Transfer</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_5' -%}
+      <div class="p-section--shallow">
+        <p>
+          On request, we transfer control over your cloud back to your organization. We train your personnel, make sure you are able to operate the cloud yourself moving forward and perform a formal handover process. We continue providing commercial support to make sure your cloud receives security updates and you receive help if needed.
+        </p>
       </div>
+    {%- endif -%}
+  {%- endcall -%}
+
+  <section class="p-section">
+    <div class="row p-section--shallow">
+      <hr class="p-rule" />
+      <h2>Private Cloud Build services</h2>
     </div>
+    <div class="col">{% include "shared/pricing/_openstack-packages.html" %}</div>
```

Comment:
You can decided if this is out of scope, but this could be converted to use the [pricing block pattern](https://vanillaframework.io/docs/patterns/pricing-block)